poster/poster.tex

```diff
@@ -0,0 +1,428 @@
+%	OBJECTIVES
+%----------------------------------------------------------------------------------------
+
+\begin{alertblock}{Objectives}
+
+Identify flexible, general, and performant algorithms available for application to simulating
+demand-driven deployment of nuclear fuel cycle facility capacity in a fuel cycle simulator.
+\begin{itemize}
+	\item Review current Nuclear Fuel Cycle simulation tools to determine their current capabilities.
+	\item Investigate promising algorithms.
+	\item Applications of such algorithms in other domains (economics, industrial engineering).  
+\end{itemize}
+
+\end{alertblock}
+
+%----------------------------------------------------------------------------------------
+%	INTRODUCTION
+%----------------------------------------------------------------------------------------
+
+\begin{block}{Introduction}
+
+\begin{figure}
+	\centering
+	\scalebox{0.8}{
+		\begin{tikzpicture}[    >=stealth,
+		node distance=6cm,
+		on grid,
+		align=center,
+		auto]
+		% Place nodes
+		\node [block] (of) {\textbf{NFC Simulation = Objective Functions}};
+		\node [cloud, below of=of] (dem) {\texttt{DEMAND} \\ ex) Achieve 1\% growth for total electricity};
+		\node [cloud, below of=dem] (con) {\texttt{CONSTRAINT} \\ex) Reprocessing available from 2025 };
+		
+		\draw[->, thick] (of) -- (dem);
+		\draw[->, thick] (dem) -- (con);
+		
+		\end{tikzpicture}
+		
+	}
+\end{figure}
+Need for : \\ Demand Repsonsive Deployment Decisions
+\begin{figure}
+	\centering
+	\scalebox{0.80}{
+		\begin{tikzpicture}[    >=stealth,
+		node distance=3cm,
+		on grid,
+		align=center,
+		auto]
+		% Place nodes
+		\node [block] (rea) {{\LARGE \texttt{Reactor}}};
+		\node [cloud, below of=rea] (d1) {1. Demand: \mathbf{Fuel}};
+		\node [cloud, below of=d1] (s1) {2. Deploy \texttt{Enrichment} Facility \\ to meet fuel demand};
+		\node [cloud, below of=s1] (ff) {3. Deploy \texttt{Mills} to meet UF6 demand};
+		\node [cloud, below of=ff] (d2) {4. Deploy \texttt{Mines} to meet uranium demand};
+		
+		
+		\draw[->, thick] (rea) -- (d1);
+		\draw[->, thick] (d1) -- (s1);
+		\draw[->, thick] (s1) -- (ff);
+		\draw[->, thick] (ff) -- (d2);
+		\end{tikzpicture}
+		
+	}
+	\caption{Dynamic Deployment to meet fuel demand}
+\end{figure}
+
+
+\end{block}
+
+\end{column} % End of the first column
+
+\begin{column}{\sepwid}\end{column} % Empty spacer column
+
+\begin{column}{\twocolwid} % Begin a column which is two columns wide (column 2)
+
+\begin{columns}[t,totalwidth=\twocolwid] % Split up the two columns wide column
+
+\begin{column}{\onecolwid}\vspace{-.6in} % The first column within column 2 (column 2.1)
+
+%----------------------------------------------------------------------------------------
+%	MATERIALS
+%----------------------------------------------------------------------------------------
+
+\begin{block}{Methods}
+	A meta-review of previous \gls{NFC} gap analyses helped to identify the existingn
```

Comment:
This throws an error : 
```! Undefined control sequence.
\beamer@doifinframe ...ta-review of previous \gls 
                                                  {NFC} gap analyses helped ...
```

That indicates to me that the poster hasn't loaded the acronyms/glossaries package.


Also, existingn -> existing